### DISCUSSION TITLE
CSS seems to not load

### DISCUSSION THREAD

**Discussion by Flapfc** [razonyang/hugo-theme-bootstrap#507] [General]
Hi, 

When using the theme following your instructions below

`$ git init`
`$ git submodule add https://github.com/razonyang/hugo-theme-bootstrap themes/hugo-theme-bootstrap`
`$ cp -a themes/hugo-theme-bootstrap/exampleSite/* .`
`$ hugo mod npm pack`
`$ npm install`
`$ hugo server`

I can get as far as building the server however when I go into local host it seems to not load any CSS within the server.

![image](https://user-images.githubusercontent.com/20688686/164224597-57b6de40-9b07-4656-90a7-ca5e9db74ab5.png)

I have checked and made sure that 

`[build]
  writeStats = true`

is present in the _default/config.toml file also.

Have I missed something as I cannot work out why the CSS would not load. I would appreciate any help you can offer.

**Comment by razonyang** [comment]
Hi, it looks like the same issue as #505. Which Hugo version do you use?

**Reply by Flapfc** [reply]
Thank you for the quick reply.

v0.97.2 extended for windows

**Reply by Flapfc** [reply]
Just moved to hugo 0.96.0 for windows instead of extended and can confirm that it is rendering correctly.

**Reply by razonyang** [reply]
Please upgrade to v0.97.3, since the patch has been reverted on v0.97.2.

**Reply by Flapfc** [reply]
I will upgrade. Thank you for the help
